docs/UserGuide.md

```diff
@@ -7,26 +7,27 @@ SWEe! is a  **desktop app for CS2103T students to manage their learning progress
 
 
 * Table of Contents
-    - Quick start
-    - Features
-        - Adding a flashcard: `add`
-        - Clear all flashcards: `clear`
-        - Deleting a flashcard: `delete`
-        - Editing a flashcard: `edit`
-        - Filter relevant flashcards: `filter`
-        - Favourite a flashcard : `fav`
-        - Unfavourite a flashcard: `unfav`
-        - Find flashcards: `find`
-        - Viewing help: `help`
-        - Listing all flashcards: `list`
-        - Review all flashcards : `review`
-        - Quiz flashcards: `quiz`
-        - Sort all flashcards: `sort`
-        - Exiting the program: `exit`
-        - View a flashcard: `view`
-        - Saving the data
+    - [Quick start](#quick-start)
+    - [Features](#features)
+        - [Adding a flashcard](#adding-a-flashcard--add): `add`
+        - [Clear all flashcards](#clearing-all-flashcards--clear): `clear`
+        - [Deleting a flashcard](#deleting-a-flashcard---delete): `delete`
+        - [Editing a flashcard](#editing-a-flashcard---edit): `edit`
+        - [Filter relevant flashcards](#filtering-out-flashcards---filter): `filter`
```

Comment:
Standardise naming. Either filter relevant or filtering out flashcards.

Reply:
Will standardise to present tense. Thank you for pointing it out.